### PR TITLE
Massive website cleanup

### PR DESCRIPTION
**Pull request by Lnaden**
This PR introduces a large number of changes, mostly filling some
content voids, stripping out remaining placeholder, and adding in some
new content where needed. I'll try to summarize in no particular order:

* Overall
    * No more faux Webp images, all true SVG, PNG, or JPG. Hopefully they will all load this time
* Team page
    * Major dev images all the same size now
    * Additional devs have a section w/o images
    * New Thanks section for non devs and contributors
    * Fixed the social/github icon links which should appear now.
* Community page
    * Uses a custom HTML page to target a custom call-to-action
    * Has community projects on it now, namely OpenFF and Distributed Driver
    * I wrote a blob about Distributed Driver which could use a look over (@loriab)
    * New custom call to action on community
* Contact
    * Added a "Contact us" mailto in the footer.
    * Contact page content updated, but not used for now. Left in place in case we decide later, unreachable other than direct url
* Homepage
    * top boxology image now centered
    * Hopefully these render now...
* License and Funding
    * Got the actual grant in
    * Removed the hard coded extra content at bottom
    * Wrangled the markdown to look better
* Use Cases
    * Moved Use Cases away from FAQ page, which no longer exists and has no links to it due to no content for it (for now)
    * Use cases is now its own HTML code to help us keep it separate from the FAQ when it comes back
    * Added a paragraph setting up the use cases

### CODE REVIEW

File: config.toml [hunk 1]
```diff
@@ -67,31 +67,37 @@ paginate = 6
...
     name = "QCA Projects"
     URL = "/projects"
+    weight = 1
 
     [[menu.main]]
     parent = "Ecosystem"
     name = "Use Cases"
-    URL = "/faq"
+    URL = "/usecases"
+    weight = 3
```
Comment (dgasmith) [inline config.toml]: Oh good! I was wondering how we sorted these things.

File: data/community.yml [hunk 2]
```diff
@@ -1,140 +1,41 @@
...
-      imageAlt : "images/people/chayafull.jpg"
-      name : "Chaya Stern"
-      designation : "Open Forcefield Power User"
-      content : "Scientist on behalf of Open Forcefield who has been using QCArchive since the beginning and helping shape end-user interface and development"
+    # Open Force field Consortium
+    - image : "images/collaborators/OpenFFVector.png"
+#      imageAlt : "images/collaborators/OpenFFVector.png"
+      name : "Open Force Field Consortium"
+      designation : "Collaborator and Sponsor"
+      content : "The <a href='url'>OpenFF Consortium</a> is an academic-industry collaboration designed to improve computer models used to
```
Comment (dgasmith) [inline data/community.yml]: https://openforcefield.org

File: data/community.yml [hunk 3]
```diff
@@ -1,140 +1,41 @@
...
+                 guide pharmaceutical drug discovery. They are a major sponsor of the QCArchive and the customer for the use case of
```
Comment (dgasmith) [inline data/community.yml]: They are definitely partners, not customers.

File: data/team.yml [hunk 4]
```diff
@@ -33,106 +32,67 @@ teamLeaders:
...
+
 
 
 # design team
 designTeam:
   enable : true
-  title : "Additional developers"
-  description : "More than three people does a project this size make! These additional developers are those who
+  title : "Additional developers and contributors"
+  description : "More than four people does a project this size make! These additional developers are those who
```
Comment (dgasmith) [inline data/team.yml]: Wording.

File: data/team.yml [hunk 5]
```diff
@@ -33,106 +32,67 @@ teamLeaders:
         - icon : "tf-ion-social-github"
           URL : "https://github.com/lnaden"
 
+    - image: "images/people/lorifull-ish.jpg"
+#      imageAlt: "images/people/lorifull-ish.webp"
+      name: "Lori Burns"
+      designation: "Lead Psi4 Developer"
+      content: "Lead Psi4 dev and developer for the QC Engine and QC Elemental projects"
```
Comment (dgasmith) [inline data/team.yml]: "Developer for the QCEngine and QCElemental projects." No spaces or abbreviations.

File: data/usecases.yml [hunk 6]
```diff
@@ -1,101 +1,113 @@
-faq:
-  - title : "Use Cases"
-    faqList :
-      - question : "I am a new student who is interested in machine learning, and I would like to obtain the ANI-1
+caseOpener:
+    enable: true
+    title: "Driven and developed by community needs"
+    content: 'The QCArchive project started from the needs wants of the quantum chemistry community and the fields which
```
Comment (dgasmith) [inline data/usecases.yml]: Wording "the needs of the".

File: data/team.yml (hunk 5 above)
Comment (loriab) [reply data/team.yml]: if you include Psi4, don't promote me beyond "a Psi4 core dev", please. and if you need a website anywhere, use my GH. it's the only thing that knows what I'm doing.

File: data/community.yml [hunk 7]
```diff
@@ -1,140 +1,41 @@
...
+      designation: "Developer and Extension"
+      content: "The Distributed Driver is a Psi4 compute driver designed to handle nested, conditional, and almost
+                programmatic operations in an iterative fashion. Multiple calculation managers and all their children
+                child managers can be staged together such that compute commands issued at parents are propagated to the
+                children.
+                This sort of distribution and job staging is exactly what the Fractal job queue and distributed
+                managers were designed to handle. Engine's schema-based input enables the type of programmatic execution
+                needed to handle these computations. Fractal, Engine, and Schema together vastly reduce the barrier to
+                efficiently setup and implement this technology and the Distributed Driver handles the top level task
+                creation logic and post-compute processing."
```
Comment (loriab) [inline data/community.yml]: Not wholly sure what the first paragraph means, but I'll believe you. the "children child managers" needs rewording, though.

**Comment by dgasmith** [comment]
Browsing the preview, it looks fantastic!

**Pull request merged**